community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -189,6 +189,14 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
     @Internal
     public static final Setting<URI> browser_path = setting( "unsupported.dbms.uris.browser", Settings.URI, "/browser/" );
 
+    @Deprecated
+    @Description( "Whether to allow executing scripts inside the server, from external sources, e.g. via traversal " +
+                  "endpoints. This setting is in the 'unsupported' namespace, because the scripting feature might be " +
```

Comment:
i think we should say that this setting will be removed in future releases, like for sure, and also ask people to use cypher or anything else instead